interfaces/builtin/linux_framebuffer.go

```diff
@@ -0,0 +1,40 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+import "github.com/snapcore/snapd/interfaces"
+
+const linuxFramebufferConnectedPlugAppArmor = `
+# Description: needed for applications using framebuffer
+
+/dev/fb0 rw,
+/dev/graphics/0 rw,
+/dev/tty0 rw, 
+/dev/tty rw,
+/dev/console rw,
+`
+
+func NewLinuxFramebufferInterface() interfaces.Interface {
+	return &commonInterface{
+		name: "linux-framebuffer",
```

Comment:
https://github.com/torvalds/linux/blob/master/Documentation/admin-guide/devices.txt refers to this as 'universal framebuffer' and I've seen it referred to as 'console framebuffer'. Here are some possible names:
1. linux-framebuffer
2. universal-framebuffer
3. console-framebuffer
4. framebuffer

I think I prefer 'console-framebuffer'. For tty1-63 we can create 'console-tty' and the naming is nice.